proof-systems-shim/src/lib.rs

```diff
@@ -17,3 +17,43 @@
 pub use mina_hasher;
 pub use mina_signer;
 pub use o1_utils;
+
+/// macro that implements hashable trait for a reference value
+#[macro_export]
```

Comment:
If we do have to include this macro I think it should live somewhere else. I think we should keep the shim crate as a pure shim (no additional code)

Reply:
Removed the macro as was not really helping much